src/CodeGen_Hexagon.cpp

```diff
@@ -62,11 +62,6 @@ std::unique_ptr<llvm::Module> CodeGen_Hexagon::compile(const Module &module) {
         if (module.target().has_feature(Halide::Target::HVX_64))
```

Comment:
The hvx_setting declared/set a few lines up from here is now un-used, can you remove it?